### DISCUSSION TITLE
Serverless Components CLI v1 is no longer bundled with Serverless Framework CLI

### DISCUSSION THREAD

**Discussion by ognitio-technologies** [serverless/serverless#10880] [Q&A]
Serverless Components CLI v1 is no longer bundled with Serverless Framework CLI 
<img width="749" alt="Screenshot 2022-03-21 at 2 50 25 PM" src="https://user-images.githubusercontent.com/72864567/159233459-aa66f9a4-2963-4a26-bca4-8365fcda7c06.png">

**Comment by pgrzesik** [comment]
Hey @ognitio-technologies - that is correct and expected as it has been removed as a part of `v3` major release. You can still use it following the instructions (installing the Components CLI v1).
